data/Yasshu55.json

```diff
@@ -0,0 +1,38 @@
+{
+	"name": "Yashwanth Sai",
+	"type": "personal",
+	"bio": "Full Stack developer, Open source enthusiast",
+	"tags": [
+		"Open Source",
+		"JavaScript"
+	],
+	"socials": [{
+			"icon": "FaTwitter",
+			"url": "https://twitter.com/Yasshu55"
+		},
+		{
+			"icon": "FaGithub",
+			"url": "https://github.com/eddiejaoude"
+		},
+		{
+			"icon": "FaYoutube",
+			"url": "https://youtube.com/eddiejaoude"
+		}
+	],
+
+	"links": [{
+			"group": "Primary Socials",
+			"name": "Twitter: Follow me",
+			"url": "https://twitter.com/eddiejaoude",
+			"icon": "FaTwitter",
+			"color": "#00ACEE"
+		},
+		{
```

Comment:
```suggestion
```

Reply:
@krshkun sorry about that, I've made the changes can you please check?